docs/use_cases.md

```diff
@@ -28,7 +28,13 @@ All users within BigUniversity need to use their existing enterprise credentials
 Alice arrives at work and logs into his desktop with his corporate username/password, and sees an email that the private cloud is now available. She clicks the link and it takes her to a sign-on page. She enters her corporate username/password and it lets her in. After exploring the features, she is ready to start using the system.  She immediately starts to query for her desired cohort.
```

Comment:
Keystone also acts as a service catalog to let other OpenStack systems know where relevant API endpoints exist for OpenStack Services.